snare/tests/test_parse_tanner_response.py

```diff
@@ -105,17 +185,5 @@ def test_call_handle_html(self):
         self.loop.run_until_complete(test())
         self.handler.html_handler.handle_content.assert_called_with(self.call_content)
 
-    def test_parse_exception(self):
```

Comment:
Why did you remove this?

Reply:
The coverage is reached using other functions so need for this one.

Reply:
I think this test is for [this](https://github.com/mushorg/snare/blob/master/snare/tanner_handler.py#L75) when detection is empty then raises a `keyError`. Since the code to handle is not written (which is because is unlikely to happen), that's why it's not present in the coverage. So I guess no harm in keeping this.